tests/templates/test_create_provider_ingester.py

```diff
@@ -0,0 +1,110 @@
+from pathlib import Path
+
+import pytest
+
+from openverse_catalog.templates import create_provider_ingester
+
+
+@pytest.mark.parametrize(
+    "media_types_str, expected_types",
+    [
+        # Just image
+        (
+            [
+                "image",
+            ],
+            [
+                "image",
+            ],
+        ),
```

Comment:
FWIW I believe if you remove the trailing comma from the entries, `black` should fit this all on one line  (same for the other entries below):
```suggestion
        (["image"], ["image"]),
```

Reply:
It's such an odd quirk of `black`, I wasn't aware of it for the longest time 😅